src/v2/services/implementations/ExtrinsicCallService.ts

```diff
@@ -0,0 +1,80 @@
+import { inject, injectable } from 'inversify';
+import { SubmittableExtrinsic } from '@polkadot/api/types';
+import { Symbols } from 'src/v2/symbols';
+import { ISubmittableResult } from '@polkadot/types/types';
+import { IExtrinsicCallService } from '../IExtrinsicCallService';
+import { IExtrinsicCallRepository } from 'src/v2/repositories/IExtrinsicCallRepository';
+import { BusyMessage, ExtrinsicStatusMessage, IEventAggregator } from 'src/v2/messaging';
+import { WalletService } from './WalletService';
+import { Guard } from 'src/v2/common';
+import { EcdsaAccount } from 'src/store/general/state';
+
+@injectable()
+export class ExtrinsicCallService extends WalletService implements IExtrinsicCallService {
```

Comment:
@impelcrypto before moving on that PR your made, just wanna check. do you think is it good for making extrinsicCallService like this?

Reply:
@sirius651 
I think we don't need to have this `extrinsicCallService` file so that we don't need to make (duplicate) another `send` method 🙂 CC @bobo-k2